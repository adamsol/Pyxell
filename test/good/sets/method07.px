
print {'a', 'c'}.has('b')
print {null}.has(null)

s = {1...1e5}
for i in -10^^5..10^^5 do
    if s.has(i) != (i >= 0) do
        print i
