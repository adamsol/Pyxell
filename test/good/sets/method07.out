false
true
0
100000
